docs/documentation/case.md

```
@@ -752,29 +752,29 @@ Details of the transducer acoustic source model can be found in [Maeda and Colon
-| `bub_pp%mu_g`†     | Real    | Viscosity of gas (bubble) |
-| `bub_pp%gam_v`†‡   | Real    | Specific heat ratio of host in vapor state |
-| `bub_pp%gam_g`*†‡  | Real    | Specific heat ratio of gas (bubble) |
-| `bub_pp%M_v`†‡     | Real    | Molecular weight of host |
-| `bub_pp%M_g`†‡     | Real    | Molecular weight of gas (bubble) |
-| `bub_pp%k_v`†‡     | Real    | Thermal conductivity of host in vapor state |
-| `bub_pp%k_g`†‡     | Real    | Thermal conductivity of gas (bubble) |
-| `bub_pp%cp_v`‡     | Real    | Specific heat capacity in constant pressure of host in vapor state |
-| `bub_pp%cp_g`‡     | Real    | Specific heat capacity in constant pressure of gas (bubble) |
-| `bub_pp%R_v`†‡     | Real    | Gas constant of host in vapor state |
-| `bub_pp%R_g`†‡     | Real    | Gas constant of gas (bubble) |
+| `bubbles_euler`    | Logical | Ensemble-averaged bubble modeling	|
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 67

(MD010, no-hard-tabs)